lib/store/src/csf/processCSFFile.ts

```diff
@@ -1,10 +1,10 @@
-import { isExportStory, sanitize, Parameters, AnyFramework, ComponentTitle } from '@storybook/csf';
+import { isExportStory, Parameters, AnyFramework, ComponentTitle } from '@storybook/csf';
 import { logger } from '@storybook/client-logger';
 
-import { ModuleExports, CSFFile, NormalizedComponentAnnotations } from './types';
+import { ModuleExports, CSFFile, NormalizedComponentAnnotations } from '../types';
 import { normalizeStory } from './normalizeStory';
-import { normalizeInputTypes } from './normalizeInputTypes';
-import { Path } from '.';
+import { Path } from '..';
```

Comment:
```suggestion
import type { Path } from '../types';
```

Isn't this a circular dependency otherwise?